doc/Command_Index_User.tex

```diff
@@ -1008,6 +1008,26 @@ \subsubsection{End of Line Behavior}
   & \texttt{disable} & The shifts are not applied.
 \end{argtable}
 
+\macroname{\textbackslash grebolshiftcleftype}{\{\#1\}}{gregoriotex-spaces.tex}
+Macro to determine how notes should be left aligned in the case where clefs of different widths appear in the same score.
+
+\begin{argtable}
+  \#1 & \texttt{largest} & The notes are aligned as if all clefs had the width of the largest clef (default)\\
+  & \texttt{current} & The notes are aligned on the current clef, which leads to unaligned notes. This was the default of Gregorio < \texttt{5.0}.
+\end{argtable}
+
+\macroname{\textbackslash grelocalbolshiftcleftype}{\{\#1\}}{gregoriotex-spaces.tex}
+Equivalent of \verb=\grebolshiftcleftype= but valid only until the next end of a score, and with more options. This can be used inside gabc for corner cases like different alignment on a score taking two pages.
+
+\begin{argtable}
+  \#1 & \texttt{largest} & same as in \verb=\grebolshiftcleftype=\\
```

Comment:
Similarly, as this is user documentation, I would repeat the descriptions here rather then refer to another macro.